scripts/gulpfiles/build_tasks.js

```diff
@@ -207,7 +207,7 @@ goog.provide('Blockly.FieldTextInput');
 goog.provide('Blockly.FieldVariable');
 goog.provide('Blockly.Mutator');
 goog.provide('Blockly.Warning');`;
-  return gulp.src(maybeAddClosureLibrary(['blocks/*.js']), {base: './'})
+  return gulp.src(['blocks/*.js'], {base: './'})
```

Comment:
Should the other calls to maybeAddClosureLibrary be removed as well?

Reply:
Nope, just this one.